### PR TITLE
doc: translate install.md

### PR DESCRIPTION
**Pull request by Qianmoxsn**
translate of install.md

### CODE REVIEW

File: docs/guide/getting_start/install.md [hunk 1]
```diff
@@ -1,67 +1,66 @@
-# 安装
+# Installation
 
-## 通过 NPM 安装
-我们推荐使用基于 `Node.js` 的前端构建工具来开发 `Web3D` 应用，比如配合 [vite](https://vitejs.dev/) 或 [webpack](https://webpack.js.org/) 等前端生态框架来构建复杂的前端项目，当然也可以更方便的使用 [TypeScript](https://www.typescriptlang.org/) 进行开发。
-1. 安装依赖
+## Install by NPM
+We recommend using frontend build tools based on Node.js to develop Web3D applications. For example, you can use frontend ecosystem frameworks such as vite or webpack to build complex frontend projects. You can also use TypeScript for more convenient development.
```
Comment (lslzl3000) [inline docs/guide/getting_start/install.md:4]: please add links for certain keywords

**Review by lslzl3000 — COMMENTED** [review]
1. check links
2. some translation is not accurate, you can check [vue doc](https://vuejs.org/guide/quick-start.html#using-vue-from-cdn) as a reference

**Review by lslzl3000 — APPROVED** [review]

**Comment by Noah133552** [comment]
After reading this article, I found some mistakes and perspectives to discuss. 
Please feel free to share ideas (many points I'm not confident LOL), and if you feel some are rational, please reconsider and rewrite the corresponding sentences. 
1. Line 11. Import modules, with a capital letter
2. The first word of a sentence should have a capital letter, and please notice the sentences after code blocks. Line 13, 17, etc.
3. Line 23. Official CDN is an example to illustrate three ways to import the Orillusion engine, so I think "here is an example of xxx CDN" is not logically correct. In other words, the three ways are not the CDN link's example.
4. Line 24. This sentence uses "A or B, etc". I feel it is a little bit awkward because etc (et cetera) means "and so on" or "and other things", so it is proper to use "A, B, or some others".

-    I'm not confident about the use of etc, so I did some research and the reference can be found at https://english.stackexchange.com/questions/137570/how-do-you-use-etc-with-or 

5. Line 24 again, in the second sentence, I suggest using "you can download and deploy relevant files locally", which can express the points of the Chinese version in a simpler way. 
6. Line 30. "Global 'Orillusion' variable" can be changed to "Global variable (named) 'Orillusion' " or "'Orillusion' (a global variable)". I think Orillusion is the name of the global variable
7. Line 30. The variable is "introduced" or "declared"? I'm not sure if "introduce a variable" is a collocation in "IT" .

-   I read the article about JS const, which usually uses "declared". https://www.w3schools.com/js/js_const.asp

8. Line 38, two mistakes

-  Recommend doing

-  For development

9. Line 46. "Of cause" is unnecessary and unprofessional in academic articles,  thus we may try to avoid using this collocation. (Just advice, because our document is not for academic purposes XD~)
10. Line 55. Shall we use the plural form of nouns? (I have no idea if there are several modules) "You can define names and corresponding addresses of the ES modules".

Some other suggestions are as follows:
11. Too many "also", and "you can". In some cases, these can be removed, for instance:
- In line 55, "You can define the name and corresponding address of the ES module" can be changed to "Define names and corresponding addresses of the ES modules"

-  In line 63, "You can use custom names to import" can be changed to "Use custom names to import"

- So many  "also" are unnecessary.

13. It is encouraged to use conjunctive adverbs to link reasons and results, which is also a problem in the Chinese version. e.g. Line 38
14. After translation, it is recommended to read the deliverables thoroughly and try to paraphrase the sentences that make you bored and confused.

**Comment by Qianmoxsn** [comment]
i've made some changes, please check it again
@Noah133552

**Review by Noah133552 — APPROVED** [review]

**Pull request merged**